components/brave_wallet_ui/page/screens/send/components/select_address_modal/select_address_modal.tsx

```diff
@@ -463,7 +463,8 @@ export const SelectAddressModal = React.forwardRef<HTMLDivElement, Props>(
                         account.accountId.uniqueKey === fromAccountId?.uniqueKey
                       }
                       accountAlias={
-                        fevmTranslatedAddresses?.[account.accountId.address]
+                        fevmTranslatedAddresses?.get(account.accountId.address)
```

Comment:
Hey could someone from @brave/crypto-wallets-front-end please verify this is correct? The type of `fevmTranslatedAddresses` is a Map but we were treating it like a record.